src/Glpi/Config/LegacyConfigurators/SessionStart.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Glpi\Config\LegacyConfigurators;
+
+use Glpi\Config\ConfigProviderHasRequestTrait;
+use Session;
+use Glpi\Config\ConfigProviderWithRequestInterface;
+use Glpi\Config\LegacyConfigProviderInterface;
+
+final class SessionStart implements LegacyConfigProviderInterface, ConfigProviderWithRequestInterface
+{
+    use ConfigProviderHasRequestTrait;
+
+    /**
+     * An array of regular expressions of the paths that disable the Session.
+     */
+    private const NO_SESSION_PATHS = [
+        '/api(rest)?\.php.*',
+        '/caldav\.php.*',
+    ];
```

Comment:
There are session usages on these paths, but they must not be sent in cookies. The constant naming and the comment should be changed to reflect this.